src/utils/__tests__/validation.spec.js

```diff
@@ -60,7 +61,18 @@ describe('validation', () => {
 		}
 	};
 
-	const validate = validation({components, paths});
+	const validate = validation({
+		components,
+		paths,
+		ajvKeywords: [{
+			name: 'mustPositive',
+			def: {
+				validate: (schema, data) => {
+					return schema ? data > 0 : true;
```

Comment:
Are you actually meaning 
```
{
  minimum: 0,
  exclusiveMinimum: true
}
```
https://swagger.io/docs/specification/data-models/data-types/#numbers

Reply:
It is just testing if customizing keyword works fine. It is meaningless, only for testing.